docs/reference/readme.md

```diff
@@ -855,26 +855,26 @@ Refresh cooldown is the minimum amount of time between allowed manually refreshe
 Do not append proxy IP address to `x-forwarded-for` HTTP header. See [Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers.html?highlight=skip_xff_append#x-forwarded-for) docs for more detail.
 
 
-## Cache Service
-The cache service is used for storing user session data.
+## Data Broker Service
```

Comment:
Just general commentary - I think we need to revisit this both in functionality and docs since the concepts have evolved.  Can be a follow-up as we will have a few areas to address separately.